client/src/main/java/academy/konrad/group/battleships/userinterface/BattleshipClient.java

```diff
@@ -26,35 +34,100 @@ void play(TextArea textArea, TilePane playerBoard, TilePane enemyBoard) {
       String fromServer;
 
       try {
-        while ((fromServer = in.readLine()) != null) {
-
-          if (fromServer.startsWith("MESSAGE")) {
-            System.out.println("Server: " + fromServer);
-            String message = fromServer;
+        while (!(fromServer = in.readLine()).equals("CLOSE")) {
```

Comment:
Fajny, prosty sposób komunikacji z serwerem. Może to jakoś de/serializować do obiektu z nagłówkiem i treścią? Byłoby bardziej obiektowo. :)